<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCustomPlot" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>621</width>
      <height>541</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="algorithms">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>570</y>
      <width>141</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="x1">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>560</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>x1:</string>
    </property>
   </widget>
   <widget class="QLabel" name="x2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>560</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>x2:</string>
    </property>
   </widget>
   <widget class="QLabel" name="y1">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>590</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>y1:</string>
    </property>
   </widget>
   <widget class="QLabel" name="y2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>590</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>y2:</string>
    </property>
   </widget>
   <widget class="QLabel" name="R">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>580</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>R:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="x1_spin">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>560</y>
      <width>62</width>
      <height>26</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-999.990000000000009</double>
    </property>
    <property name="maximum">
     <double>999.990000000000009</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="y1_spin">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>590</y>
      <width>62</width>
      <height>26</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-999.990000000000009</double>
    </property>
    <property name="maximum">
     <double>999.990000000000009</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="x2_spin">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>560</y>
      <width>62</width>
      <height>26</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-999.990000000000009</double>
    </property>
    <property name="maximum">
     <double>999.990000000000009</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="y2_spin">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>590</y>
      <width>62</width>
      <height>26</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-999.990000000000009</double>
    </property>
    <property name="maximum">
     <double>999.990000000000009</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="R_spin">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>580</y>
      <width>62</width>
      <height>26</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-999.990000000000009</double>
    </property>
    <property name="maximum">
     <double>999.990000000000009</double>
    </property>
   </widget>
   <widget class="QPushButton" name="paintButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>570</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>нарисовать</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>620</y>
      <width>281</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>651</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
